generators/spa/templates/VueJS/common/Scripts/QuickSettings.js

```diff
@@ -1,32 +1,97 @@
-var <%= moduleName %>Settings = <%= moduleName %>Settings || {};
+var dnnspamodule = dnnspamodule || {};
```

Comment:
@VAlmea We can't save this the way it is right now.  If I build two modules using the same template with this update, they will break every time I put them on the same page.  This is why the variable name was part of the template before.  If the module is added to the page more than once, there won't be a JS error related to this object name.

Reply:
@VAlmea This also was never addressed.

Reply:
Fixed